Python_Script.yml

```diff
@@ -0,0 +1,48 @@
+import requests,json, yaml
+
+out_file='Open_Source_output.yml'
+d={}
+source_yaml_data=input("Please Enter the source yaml file:")
```

Comment:
You should not take input expect this file is already available in repository

Reply:
Included the input file into the script